### PR TITLE
Adding many more "echo $variable" statements

### PR DESCRIPTION
**Pull request by skabou**
These changes fixes: #17
- Added a couple spots to check variables before running a `sed` command
- Updated the GitHub username `sed` update to be idempotent

### CODE REVIEW

File: docs/deploy/06-aks-cluster.md [hunk 1]
```diff
@@ -25,17 +25,17 @@ Following the steps below will result in the provisioning of the AKS multi clust
 1.  Obtain shared services resource details
 
     ```bash
-    LOGANALYTICSWORKSPACEID=$(az deployment group show -g rg-bu0001a0042-shared -n shared-svcs-stamp --query properties.outputs.logAnalyticsWorkspaceId.value -o tsv)
-    CONTAINERREGISTRYID=$(az deployment group show -g rg-bu0001a0042-shared -n shared-svcs-stamp --query properties.outputs.containerRegistryId.value -o tsv)
+    export LOGANALYTICSWORKSPACEID_AKS_MRB=$(az deployment group show -g rg-bu0001a0042-shared -n shared-svcs-stamp --query properties.outputs.logAnalyticsWorkspaceId.value -o tsv)
+    export CONTAINERREGISTRYID_AKS_MRB=$(az deployment group show -g rg-bu0001a0042-shared -n shared-svcs-stamp --query properties.outputs.containerRegistryId.value -o tsv)
```
Comment (ckittel) [inline docs/deploy/06-aks-cluster.md:29]: Why?

export + `_AKS_MRB` should only be used on env variables that are used on subsequent pages.  That's been the model in all the other AKS baselines.  If it's on the same page, that's not captured.  We only persist where necessary (spans pages).

File: docs/deploy/06-aks-cluster.md [hunk 2]
```diff
@@ -25,17 +25,17 @@ Following the steps below will result in the provisioning of the AKS multi clust
...
 
 1.  Get the corresponding AKS cluster spoke VNet resource IDs for the app team working on the application A0042.
 
     > :book: The app team will be deploying to a spoke VNet, that was already provisioned by the network team.
 
     ```bash
-    RESOURCEID_VNET_BU0001A0042_03=$(az deployment group show -g rg-enterprise-networking-spokes -n spoke-BU0001A0042-03 --query properties.outputs.clusterVnetResourceId.value -o tsv)
-    RESOURCEID_VNET_BU0001A0042_04=$(az deployment group show -g rg-enterprise-networking-spokes -n spoke-BU0001A0042-04 --query properties.outputs.clusterVnetResourceId.value -o tsv)
+    export RESOURCEID_VNET_BU0001A0042_03_AKS_MRB=$(az deployment group show -g rg-enterprise-networking-spokes -n spoke-BU0001A0042-03 --query properties.outputs.clusterVnetResourceId.value -o tsv)
+    export RESOURCEID_VNET_BU0001A0042_04_AKS_MRB=$(az deployment group show -g rg-enterprise-networking-spokes -n spoke-BU0001A0042-04 --query properties.outputs.clusterVnetResourceId.value -o tsv)
```
Comment (ckittel) [inline docs/deploy/06-aks-cluster.md:38]: Why?

**Review by ckittel — CHANGES_REQUESTED** [review]
Also, can you please apply the same experience that's in the AKS baseline with the echos after setting.

File: docs/deploy/06-aks-cluster.md (hunk 1 above)
Comment (skabou) [reply docs/deploy/06-aks-cluster.md:29]: I've run into issues where I ran the big `sed` command only to later find some variables were empty. But now that there's the part that echos out the variables then yes perhaps adding these to the environment variables may be unnecessary.

Comment (skabou) [reply docs/deploy/06-aks-cluster.md:29]: Will check out AKS baseline for echos

Comment (ckittel) [reply docs/deploy/06-aks-cluster.md:29]: Yeah, and doesn't use what the `_AKS_MRB` is designed for -- which is checkpointing at the end of a page.

File: docs/deploy/01-prerequisites.md [hunk 3]
```diff
@@ -44,6 +44,8 @@ This is the starting point for the instructions on deploying the [AKS baseline m
 
    ```bash
    export GITHUB_USER_NAME_AKS_MRB=$(echo $(gh auth status 2>&1) | sed "s#.*as \(.*\) (.*#\1#")
+   echo GITHUB_USER_NAME_AKS_MRB: $GITHUB_USER_NAME_AKS_MRB
+
```
Comment (ckittel) [inline docs/deploy/01-prerequisites.md:48]:
```suggestion
```

File: docs/deploy/02-aad.md [hunk 4]
```diff
@@ -62,16 +64,25 @@ This does not configure anything related to workload identity. This configuratio
...
+   echo AADOBJECTNAME_GROUP_CLUSTERADMIN_BU0001A004203: $AADOBJECTNAME_GROUP_CLUSTERADMIN_BU0001A004203
+   echo AADOBJECTNAME_GROUP_CLUSTERADMIN_BU0001A004204: $AADOBJECTNAME_GROUP_CLUSTERADMIN_BU0001A004204
+   echo AADOBJECTID_GROUP_CLUSTERADMIN_BU0001A004203_AKS_MRB: $AADOBJECTID_GROUP_CLUSTERADMIN_BU0001A004203_AKS_MRB
+   echo AADOBJECTID_GROUP_CLUSTERADMIN_BU0001A004204_AKS_MRB: $AADOBJECTID_GROUP_CLUSTERADMIN_BU0001A004204_AKS_MRB
+   
    # assign the admin as new member in both groups
    az ad group member add -g $AADOBJECTID_GROUP_CLUSTERADMIN_BU0001A004203_AKS_MRB --member-id $AADOBJECTID_USER_CLUSTERADMIN
    az ad group member add -g $AADOBJECTID_GROUP_CLUSTERADMIN_BU0001A004204_AKS_MRB --member-id $AADOBJECTID_USER_CLUSTERADMIN
+   
+
```
Comment (ckittel) [inline docs/deploy/02-aad.md:85]:
```suggestion
```

File: docs/deploy/06-aks-cluster.md [hunk 5]
```diff
@@ -89,6 +94,18 @@ Following the steps below will result in the provisioning of the AKS multi clust
 
     1.  Generate cluster parameter file per region
 
+        Verify all variables are populated:
```
Comment (ckittel) [inline docs/deploy/06-aks-cluster.md:97]:
```suggestion
        Verify all variables are populated:

```

File: docs/deploy/06-aks-cluster.md [hunk 6]
```diff
@@ -89,6 +94,18 @@ Following the steps below will result in the provisioning of the AKS multi clust
...
+        echo RESOURCEID_VNET_BU0001A0042_03: $RESOURCEID_VNET_BU0001A0042_03
+        echo RESOURCEID_VNET_BU0001A0042_04: $RESOURCEID_VNET_BU0001A0042_04
+        echo TENANTID_K8SRBAC_AKS_MRB: $TENANTID_K8SRBAC_AKS_MRB
+        echo AADOBJECTID_GROUP_CLUSTERADMIN_BU0001A004203_AKS_MRB: $AADOBJECTID_GROUP_CLUSTERADMIN_BU0001A004203_AKS_MRB
+        echo AADOBJECTID_GROUP_CLUSTERADMIN_BU0001A004204_AKS_MRB: $AADOBJECTID_GROUP_CLUSTERADMIN_BU0001A004204_AKS_MRB
+        echo LOGANALYTICSWORKSPACEID: $LOGANALYTICSWORKSPACEID
+        echo CONTAINERREGISTRYID: $CONTAINERREGISTRYID
+        ```
+        
+        Update each region's cluster parameter file:
```
Comment (ckittel) [inline docs/deploy/06-aks-cluster.md:108]:
```suggestion
        Update each region's cluster parameter file:

```

**Review by ckittel — APPROVED** [review]
Just these extra spaces, otherwise, looks good!

**Pull request merged**